rpc-client-api/src/response.rs

```diff
@@ -424,6 +424,7 @@ pub struct RpcSimulateTransactionResult {
     pub units_consumed: Option<u64>,
     pub return_data: Option<UiTransactionReturnData>,
     pub inner_instructions: Option<Vec<UiInnerInstructions>>,
+    pub blockhash: Option<RpcBlockhash>,
```

Comment:
I would like this to indicate that this is the blockhash that replaced the transaction's recent blockhash. I think "replacement" blockhash is best but maybe "latest" blockhash could work too
```suggestion
    pub replacement_blockhash: Option<RpcBlockhash>,
```